### PR TITLE
Update lbry-basics.md

### PR DESCRIPTION
**Pull request by ceoger**
updated images wit new redesign

### Fixes #

### CODE REVIEW

File: content/faq/lbry-basics.md [hunk 1]
```diff
@@ -47,28 +45,24 @@ LBRY is a decentralized peer to peer protocol, meaning there are no centralized
...
+### Where can I see my `1`Downloaded, `2`Published items and `3`History?
+Click My LBRY on the left side of the LBRY App, then click `1`downloads to view downloaded files, `2`Published tab to view your published content and `3`to view watch or access history
+![published](https://spee.ch/5/download-publish-history.jpeg)
 
 ### How do I know if I'm sharing content and helping the LBRY network properly?
 The easiest way to confirm that you are sharing correctly is to determine if the port used for seeding (3333) is open to the rest of the LBRY network. To do so, type 3333 into [this port checking tool](http://www.canyouseeme.org) and check the result. It if shows "open", you are all set. If it shows "closed", you may need to check your router settings for UPnP options (set to enable) or forward TCP port 3333 and UDP Port 4444 to your local computer running the LBRY app. Firewall and NAT settings may also affect the availability of these ports. 
 
 ### How can I search for content on LBRY?
 Searching in LBRY is as easy as typing your search term(s) into the address bar at the top and waiting for the results to return (**don't click Enter!**).
```
Comment (tzarebczan) [inline content/faq/lbry-basics.md]: If you click enter now, you are giving both the direct url and search results...can we explain that?

File: content/faq/lbry-basics.md [hunk 2]
```diff
@@ -47,28 +45,24 @@ LBRY is a decentralized peer to peer protocol, meaning there are no centralized
 ### I want to share my content, where do I start? {#publish}
 Please refer to our [publishing guide](https://lbry.io/faq/how-to-publish) as a reference to assist you through the publishing process. 
 
-### Where can I see my Downloaded and Published items?
-Click My LBRY on the left side of the LBRY App, then click downloads to view downloaded files.
-![download](https://spee.ch/8/downloads.jpeg)
-
-Click the Published tab to view your published content.
-![published](https://spee.ch/9/pub.jpeg)
+### Where can I see my `1`Downloaded, `2`Published items and `3`History?
```
Comment (tzarebczan) [inline content/faq/lbry-basics.md]: These will all be top level in 0.28, can you take a screenshot from the RC?

File: content/faq/lbry-basics.md (hunk 1 above)
Comment (ceoger) [reply content/faq/lbry-basics.md]: I’ll have a look at the RC later today 😃

**Comment by tzarebczan** [comment]
Thanks @ceoger !!

**Pull request merged**